pkgs/applications/version-management/gitlab/update.py

```diff
@@ -17,7 +17,20 @@
 
 # Always keep this in sync with the GitLaab version you're updating to.
 # If you see any errors about vendored dependencies during an update, check the Gemfile.
-VENDORED_GEMS = ['devise-pbkdf2-encryptable', 'omniauth-cas3', 'omniauth-gitlab', 'omniauth_crowd', 'mail-smtp_pool', 'ipynbdiff', 'error_tracking_open_api']
+VENDORED_GEMS = [
+    'bundler-checksum',
+    'devise-pbkdf2-encryptable',
+    'omniauth-azure-oauth2',
+    'omniauth-cas3',
+    'omniauth-gitlab',
+    'omniauth-google-oauth2',
+    'omniauth_crowd',
+    'omniauth-salesforce',
+    'mail-smtp_pool',
+    'microsoft_graph_mailer',
+    'ipynbdiff',
+    'error_tracking_open_api',
+]
```

Comment:
Was this done by an automated formatter? (I'll probably make a PR to reformat it with black honestly, so just curious.)

Reply:
I reformatted this line by myself because it was starting to become really long. I have now reformatted that part to match black's coding style.